shared/templates/static/ansible/no_direct_root_logins.yml

```diff
@@ -0,0 +1,11 @@
+# platform = multi_platform_rhel,multi_platform_fedora
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+- name: "Direct root Logins Not Allowed"
+  shell: echo > /etc/securetty
+  args:
+    executable: /bin/bash
+  tags:
```

Comment:
Using `shell` in ansible is frowned upon and I think we can do this without `shell`.